### PR TITLE
ISPN-13849 shutdown cli help text

### PR DESCRIPTION
**Pull request by oraNod**
https://issues.redhat.com/browse/ISPN-13849

### CODE REVIEW

**Review by wfink — APPROVED** [review]

File: cli/src/main/java/org/infinispan/cli/commands/rest/Shutdown.java [hunk 1]
```diff
@@ -78,7 +78,7 @@ protected CompletionStage<RestResponse> exec(ContextAwareCommandInvocation invoc
       }
    }
 
-   @CommandDefinition(name = "container", description = "Shuts down the container without terminating the server processes", activator = ConnectionActivator.class)
+   @CommandDefinition(name = "container", description = "Stops the data container without terminating the server process. Intended for use with the Kubernetes Operator only.", activator = ConnectionActivator.class)
```
Comment (ryanemerson) [inline cli/src/main/java/org/infinispan/cli/commands/rest/Shutdown.java:81]: "Intended for use with the Kubernetes Operator only".

That's too far, as the Operator doesn't even use the CLI command. There might be some other k8 workloads where this CLI command makes sense. I would just remove this last sentence.

Comment (oraNod) [reply cli/src/main/java/org/infinispan/cli/commands/rest/Shutdown.java]: Thanks, @ryanemerson I've removed that and reworded the manpage. I think Wolf wanted to have the same hint in both places but it's too much info to fit into the `-h` output.

File: cli/src/main/resources/help/shutdown.adoc [hunk 2]
```diff
@@ -26,11 +26,12 @@ Stops the server to which the CLI is connected.
...
 `shutdown cluster` +
-Stores cluster state, persists entries if you use a cache store, and stops all nodes.
+Stops all nodes in the cluster after storing cluster state and persisting entries if there is a cache store.
 
 `shutdown container` +
-Stores cluster state and persists entries if you use a cache store. The servers remain running with active endpoints
-and clustering, however REST calls to container resources will result in a 503 Service Unavailable response.
+Stops the data container without terminating the server process. Stores cluster state and persists entries if there is a cache store. Server instances remain running with active endpoints and clustering. REST calls to container resources will result in a 503 Service Unavailable response.
++
+The `shutdown container` command is not intended for use with cluster administration. If you stop the data container you should use the `shutdown cluster` command to stop the cluster before restarting it.
```
Comment (ryanemerson) [inline cli/src/main/resources/help/shutdown.adoc:34]: This isn't right. Executing `shutdown cluster` after `shutdown container` will result in a 503 as it's trying to perform actions on the container resource which we have just shutdown. Also, the purpose of the command is for cluster administration, albeit it probably won't apply to most users.

I think we should just state something like: "The `shutdown container` command is intended for environments, such as Kubernetes, that manage resource lifecycles independently. Unless you explicitly want server processes to remain, without an active container, then it's likely that you should use the `shutdown cluster` command instead."

Comment (oraNod) [reply cli/src/main/resources/help/shutdown.adoc]: I think you've nailed it. Cheers @ryanemerson

File: cli/src/main/resources/help/shutdown.adoc [hunk 3]
```diff
@@ -29,7 +29,7 @@ Stops the server with hostname `my_server01`.
 Stops all nodes in the cluster after storing cluster state and persisting entries if there is a cache store.
 
 `shutdown container` +
-Stops the data container without terminating the server process. Stores cluster state and persists entries if there is a cache store. Server instances remain running with active endpoints and clustering. REST calls to container resources will result in a 503 Service Unavailable response. The `shutdown container` command is intended for environments, such as Kubernetes, that automate resource lifecycle management. You should always use either the `shutdown server` or `shutdown cluster` command instead.
+Stops the data container without terminating the server process. Stores cluster state and persists entries if there is a cache store. Server instances remain running with active endpoints and clustering. REST calls to container resources will result in a 503 Service Unavailable response. The `shutdown container` command is intended for environments, such as Kubernetes, that automate resource lifecycle management. For self-managed environments you should use the `shutdown server` or `shutdown cluster` commands to stop servers.
```
[the remark this answers is not in the document]
Comment (ryanemerson) [reply cli/src/main/resources/help/shutdown.adoc:33]: Looks great now :+1:

**Comment by ryanemerson** [comment]
Thanks @oraNod

**Pull request merged**